README.md

```diff
@@ -21,16 +21,31 @@ These are important benefits:
    source-level diff, LLpatch uses LLVM IR for diffing to abstract away all the
    specifics/details on machine architecture, types of linkers, and kernel version.
 
-2. **Slim Livepatch**: LLpatch uses 'llvm-diff' to perform the diffing and generates LLVM IR
+2. **Easy Callbacks and LLPatch Symbols**: Kernel livepatch allows four callbacks
```

Comment:
Is it "LLpatch" or "LLPatch"? We should be consistent.

Reply:
Let's use LLpatch. This is the only place for LLPatch. ;)